lib/zip.ts

```diff
@@ -41,7 +41,14 @@ export const getFileStreamFromZipStream = async (
 					}
 					entry.on('end', () => {
 						// Stop reading the zip archive once the file we want has been extracted.
-						zipStream.unpipe(unzipper);
+						if (zipStream.unpipe !== undefined) {
+							zipStream.unpipe(unzipper);
+						}
```

Comment:
You can do this in one:
```suggestion
						zipStream.unpipe?.(unzipper);
```